GUIDE.md

```diff
@@ -160,4 +160,17 @@ nvm use 20
 ### Install snarkjs
 ```sh
 npm install -g snarkjs
-```
\ No newline at end of file
+```
+
+
+## FAQ
+
```

Comment:
Several more FAQ:
1. what should I do when the log frequently shows "i/o deadline reached" during data syncing?
2. what does it means when the log shows "The nonces are exhausted in this slot"
3. How do I know whether I've got a mining reward? (1. dashboard 2. related log 3. miner address balance)